implementations/rust/ockam/ockam_api/src/nodes/service/tcp_inlets/node_manager_worker.rs

```diff
@@ -2,13 +2,13 @@ use ockam::{route, Result};
 use ockam_core::api::{Error, Response};
 use ockam_node::Context;
 
-use crate::nodes::models::portal::{CreateInlet, InletStatus};
+use crate::nodes::models::portal::{CreateInlet, InletStatus, InletStatusList};
 use crate::nodes::NodeManagerWorker;
 
 impl NodeManagerWorker {
-    pub(crate) async fn get_inlets(&self) -> Result<Response<Vec<InletStatus>>, Response<Error>> {
+    pub(crate) async fn get_inlets(&self) -> Result<Response<InletStatusList>, Response<Error>> {
```

Comment:
This kind of change (vec<T> => TList) is now needed because we don't have a default impl right?

Reply:
Yes. Since `Decodable` is abstract, you can't say something like "reuse the same decodable implementation to decode a `Vec`, than the one you used to decode the `T` inside the `Vec`".